spec/support/account_signup_helper_spec.rb

```diff
@@ -0,0 +1,22 @@
+module AccountSignupHelper
+  def self.test_ec_key_fixture
+    "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIBjlMYhN/1FiZX7RRUyKpqDM2bPIHQswhjm+ZPaOXlgQoAoGCCqGSM49\nAwEHoUQDQgAE2pGXU3YkENblp+SMC8CWpkwPDdWXJdEsXoSQ8dWNYxFFbnXKN4Vv\nRGeLYoNKjuK/2t9rlEEumIzm5EtMxts7Jg==\n-----END EC PRIVATE KEY-----\n"
```

Comment:
Yeah I think it does, It' a bit of a pain becuase after putting this in here i get loads of emails from security bots that want to sell us services being like "you fool, you've committed a secret!"

They issue here is the mehthod we're using to generate a key we can use to sign the JWT:
`OpenSSL::PKey::EC.new`

I've tested that with various strings:
From a
- `OpenSSL::PKey::EC.new("")`
- `OpenSSL::PKey::EC.new("test")`
- even tried `OpenSSL::PKey::EC.new
("-----BEGIN EC PRIVATE KEY-----000000000000000000000000000000000000000000000000000000000000000000000000000000-----END EC PRIVATE KEY-----)`

All of them get me the following error:
`OpenSSL::PKey::ECError (invalid curve name)`
So i figured it was better to create a new one.

Reply:
:thinking: actually though, I don't think i need this to be static anymore because i'm not trying to match to a JWT...
So ok yeah I can switch this to a new randomly generated key!

Reply:
amended that [here](https://github.com/alphagov/finder-frontend/pull/2207/commits/97d4a2e9337c9a47a6f4f209b831762fb3911714)